### PR TITLE
feat: intention management

### PR DESCRIPTION
**Pull request by 530051970**
Fixes #

### CODE REVIEW

File: source/lambda/intention/text.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        print(f'=======excel_file is {excel_file.getvalue()[:1000]}')  # 
+
+        # 使用 openpyxl 加载 Excel 文件
+        workbook = load_workbook(filename="./qalist_0822.xlsx")
+
+        # 检查工作簿中的工作表
+        sheet_names = workbook.sheetnames
+        logger.info(f"Sheet names: {sheet_names}")
+
+        # 获取第一个工作表并读取内容
```
Comment (NingLu) [inline source/lambda/intention/text.py:37]: Use English comments

File: source/lambda/intention/text.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    try:
+        # 从 S3 获取文件内容
+        response = s3_client.get_object(Bucket=bucket_name, Key=file_key)
+        file_content = response['Body'].read()
+        print(file_content[:1000])  
+        logger.info(f"File content length: {len(file_content)}")
+
+        # 使用 BytesIO 将文件内容作为字节流加载
+        excel_file = BytesIO(file_content)
+        print(f'=======type is {type(excel_file)}')
```
Comment (NingLu) [inline source/lambda/intention/text.py:27]: Use logger instead of print

File: source/lambda/intention/text.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# 配置日志记录器
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def excel():
+    # 初始化 S3 客户端
+    s3_client = boto3.client('s3')
+
+    # 定义 S3 存储桶名称和文件的路径（Key）
+    bucket_name = 'ai-customer-service-apiconstructllmbotdocumentsfc4-kq7pcgbahxwk'
```
Comment (NingLu) [inline source/lambda/intention/text.py:15]: Do not expose your bucket name to avoid getting a hijacking

File: source/portal/src/types/index.ts [hunk 4]
```diff
@@ -6,16 +6,17 @@ type ResponseConfig = {
 
 export type LibraryListItem = {
   s3Prefix: string;
-  offline: string;
+  offline: string; // 一直为true？？？
   s3Bucket: string;
   executionId: string;
   executionStatus: string;
-  qaEnhance: string;
+  qaEnhance: string; //？？？？
```
Comment (NingLu) [inline source/portal/src/types/index.ts:13]: Remove the comments, L9, L13, L16, L17

File: source/portal/src/types/index.ts [hunk 5]
```diff
@@ -24,6 +25,29 @@ export type LibraryListResponse = {
...
 
+export type IntentionsItem = {
+  // s3Prefix: string; // "intentions/Admin/testindeto.xlsx"
+  // offline: string;  // true
+  // s3Bucket: string; // intelli-agent-apiconstructllmbotdocumentsfc4f8a7a-6vbr3vihybqs
+  executionId: string;
+  executionStatus: string;
+  // operationType: string; // create
+  // sfnExecutionId: string; // 99041951-b0c5-4b39-9efa-fcee12f751c0
+  // indexType: string; // 需要吗？intention 需要改job代码
```
Comment (NingLu) [inline source/portal/src/types/index.ts:36]: Remove the comments

File: source/lambda/intention/text.py (hunk 2 above)
Comment (530051970) [reply source/lambda/intention/text.py:27]: unused file, removed！

File: source/lambda/intention/text.py (hunk 3 above)
Comment (530051970) [reply source/lambda/intention/text.py:15]: unused file, removed！

File: source/lambda/intention/text.py (hunk 1 above)
Comment (530051970) [reply source/lambda/intention/text.py:37]: unused file, removed！

File: source/portal/src/types/index.ts (hunk 4 above)
Comment (530051970) [reply source/portal/src/types/index.ts:13]: done

File: source/portal/src/types/index.ts (hunk 5 above)
Comment (530051970) [reply source/portal/src/types/index.ts:36]: done

**Pull request merged**